spec/hive_metastore_spec.rb

```diff
@@ -8,21 +8,31 @@
         stub_command('update-alternatives --display hive-conf | grep best | awk \'{print $5}\' | grep /etc/hive/conf.chef').and_return(false)
       end.converge(described_recipe)
     end
+    pkg = 'hive-metastore'
 
-    it 'install hive-metastore package' do
-      expect(chef_run).to install_package('hive-metastore')
+    it "does not install #{pkg} package" do
+      expect(chef_run).not_to install_package(pkg)
     end
 
-    it 'creates /etc/init.d/hive-metastore from template' do
+    it "runs package-#{pkg} ruby_block" do
+      expect(chef_run).to run_ruby_block("package-#{pkg}")
+    end
+
+    it "creates #{pkg} service resource, but does not run it" do
+      expect(chef_run).to_not disable_service(pkg)
+      expect(chef_run).to_not enable_service(pkg)
+      expect(chef_run).to_not reload_service(pkg)
+      expect(chef_run).to_not restart_service(pkg)
+      expect(chef_run).to_not start_service(pkg)
+      expect(chef_run).to_not stop_service(pkg)
```

Comment:
We don't. We care that the code does what it is supposed to do. In this case, we want the code to do `:nothing` but there's no ChefSpec match for `:nothing` so we have to check for not $any_valid_action_other_than_nothing ... annoying, but "correct"